node/Release/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/groovy
+@Library('github.com/fabric8io/osio-pipeline@master')_
```

Comment:
`@Library('github.com/fabric8io/osio-pipeline@master') _` 

Add a space between `) _`

Reply:
it works without the space too, any reason why it is required.

Reply:
yes, just like `def a=b+c*d*(f-g)` does but that isn't a good reason to not add space.